Libs/MRML/Core/vtkMRMLAbstractLayoutNode.cxx

```diff
@@ -0,0 +1,66 @@
+/*==============================================================================
+
+  Copyright (c) Laboratory for Percutaneous Surgery (PerkLab)
+  Queen's University, Kingston, ON, Canada. All Rights Reserved.
+
+  See COPYRIGHT.txt
+  or http://www.slicer.org/copyright/copyright.txt for details.
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+
+  This file was originally developed by Csaba Pinter, PerkLab, Queen's University
+  and was supported through CANARIE�s Research Software Program, and Cancer
+  Care Ontario.
+
+==============================================================================*/
+
+// MRML includes
+#include "vtkMRMLAbstractLayoutNode.h"
+
+//----------------------------------------------------------------------------
+vtkMRMLAbstractLayoutNode::vtkMRMLAbstractLayoutNode()
+{
+  this->HideFromEditors = 1;
+}
+
+//----------------------------------------------------------------------------
+vtkMRMLAbstractLayoutNode::~vtkMRMLAbstractLayoutNode()
+{
+}
+
+//----------------------------------------------------------------------------
+void vtkMRMLAbstractLayoutNode::WriteXML(ostream& of, int nIndent)
+{
+  Superclass::WriteXML(of, nIndent);
+}
+
+//----------------------------------------------------------------------------
+void vtkMRMLAbstractLayoutNode::ReadXMLAttributes(const char** atts)
+{
+  int disabledModify = this->StartModify();
```

Comment:
We don't need Start/EndModify now. We can add them when we add reading of some values.

Reply:
I think they make it easy to write these functions later. What aobut commenting them out if you don't want them to execute?

Reply:
Yes, commenting out would be better. It's just better to have less code in general, but of course it means extra work to put it back when you need that code, so we need a good balance (we could event take out the methods completely).